src/test/msgr/test_msgr.cc

```diff
@@ -2196,6 +2226,35 @@ TEST_P(MessengerTest, SyntheticInjectTest4) {
   g_ceph_context->_conf.set_val("ms_inject_delay_max", "0");
 }
 
+// This is test for network block, means ::send return EAGAIN
+TEST_P(MessengerTest, SyntheticInjectTest5) {
+  SyntheticWorkload test_msg(1, 8, GetParam(), 100,
+                             Messenger::Policy::stateful_server(0),
+                             Messenger::Policy::lossless_client(0),
+			     64, 2);
+  bool simulate_network_congestion = true;
+  for (int i = 0; i < 2; ++i) {
+    if (!(i % 10)) lderr(g_ceph_context) << "seeding connection " << i << dendl;
+    test_msg.generate_connection();
+  }
+  for (int i = 0; i < 5000; ++i) {
+    if (!(i % 10)) {
+      lderr(g_ceph_context) << "Op " << i << ": " << dendl;
```

Comment:
IMHO, `lderr()` is only for logging errors.

Reply:
do nothing because I remove this line :p

Reply:
hmm, just make the same with the above logging method.

here logging the statement, using normal `ldout` would be fine?